18-additional-resources.Rmd

```diff
@@ -94,12 +96,24 @@ Bryan, J. & Hestor, J. *Happy git and github for the useR*. Retrieved from [http
 
 *Introduction to dplyr*. Retreived from [https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html](https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html)
 
+> `dplyr` is a R package that helps to manipulate and understand data. It provides simple “verbs”, functions that correspond to the most common data manipulation tasks, to help you translate your thoughts into code.Anyone with no programming experience can easily understand the verbs and use them for data analytics with ease.
+
 *A short introduction to the caret package*. Retrieved from [https://cran.r-project.org/web/packages/caret/vignettes/caret.html]([https://cran.r-project.org/web/packages/caret/vignettes/caret.html)
 
+> The `caret` package is an abbreviated form of `Classification And REgression Trees`. It contains functions to streamline the model training process for complex regression and classification problems. The idea behind this package is to streamline the process for creating predictive models. The package contains tools for:
+
+    - data splitting
+    - pre-processing
+    - feature selection
+    - model tuning using resampling
+    - variable importance estimation
+
 *tidy data*. Retrieved from [https://tidyr.tidyverse.org/articles/tidy-data.html](https://tidyr.tidyverse.org/articles/tidy-data.html)
 
 Wickham et al. (2019). Welcome to the Tidyverse. *Journal of Open Source Software, 4*(43). 1686-1691. https://joss.theoj.org/papers/10.21105/joss.01686
 
+> The philosophy of ensuring data is clean such that each column contains data of just one type (or atomicity) and each row is an observation (made up of several atomic columns) is made possible in this paper. The principles of tidy data provide a standard way to organise data values within a dataset.
```

Comment:
Tis is great - is there any chance you would be willing to add a bit more on the idea of atomicity? minorly concerned that we don't (I think) mention this earlier in the book and so it may surprise books. Perhaps one way to do this is to add just a bit more but tolink to a source describing these (advanced R? R for data science?).

Reply:
thank you for the feedback. I have updated the text.